eng/config/PublishData.json

```diff
@@ -198,15 +198,23 @@
       "version": "3.8.*",
       "vsix": null,
       "packageFeeds": "default",
-      "channels": [ "dev16.8p4" ],
+      "channels": [ "dev16.8p5" ],
       "vsBranch": "rel/d16.8",
       "vsMajorVersion": 16
     },
+    "release/dev16.9-preview1-vs-deps": {
```

Comment:
Weren't we going to just a branch called release/dev16.9 for the entire cycle rather than forking this all the time?

Reply:
I just follow the instructions in the doc. 
I am also puzzled because look the 16.8 branches, it has dev16.8-preview1 and dev16.8. 
Might better to sync within the infrastructure to see which name is good

Reply:
Yeah, I expected the new shipping branch here to be named `release/dev16.9-vs-deps`. Could you please link the doc you used?

Reply:
I was following the 'Snapping new minor version section'. 
OK, the let me do a renaming for all the newly created branch

Reply:
Done, mark as resolved.